structuralcodes/geometry/_geometry.py

```diff
@@ -82,9 +82,9 @@ def return_global_counter_and_increase(cls):
 
     @staticmethod
     def from_geometry(
-        geo: Geometry,
+        geo: 'Geometry',
```

Comment:
now that you restored future, can we use `Geometry` instead of `'Geometry'`?

Reply:
Sure 👍  fixed in be08c1d